Makefile

```diff
@@ -63,6 +65,15 @@ endif
 .PHONY: build
 build: checkenv build-adapter build-controller
 
+.PHONY: release
```

Comment:
I'd move `Release` section to right after the `Publish` to keep the `Build` one consistent (line ~53). And could you please add header/comments to that section as well, just for the consistency :) What I mean:

```
##################################################
# Release                                                                                            #
##################################################
```

Reply:
Sounds good :)